### PR TITLE
Fragment sign

### PR DESCRIPTION
**Pull request by Antonio06188**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/healthplay/auth/SignedInFragment.java [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        view = inflater.inflate(R.layout.fragment_signed_in, container, false);
+        IdpResponse response = getActivity().getIntent().getParcelableExtra(IDP_RESPONSE);
+        FirebaseUser currentUser = FirebaseAuth.getInstance().getCurrentUser();
+        if (currentUser == null) {
+            startActivity(AuthUiActivity.createIntent(getActivity()));
+            //finish();
+            return view;
```
Comment (Tetardfurieux) [inline app/src/main/java/ch/epfl/sdp/healthplay/auth/SignedInFragment.java:75]: Please remove dead code

File: app/src/main/java/ch/epfl/sdp/healthplay/auth/SignedInFragment.java [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+    }
+
+    public void signOut() {
+        AuthUI.getInstance()
+                .signOut(getContext())
+                .addOnCompleteListener(task -> {
+                    if (task.isSuccessful()) {
+                        startActivity(new Intent(getContext(), HomeScreenActivity.class));
+                        //finish();
+                        return;
```
Comment (Tetardfurieux) [inline app/src/main/java/ch/epfl/sdp/healthplay/auth/SignedInFragment.java:132]: remove the comment

File: app/src/main/java/ch/epfl/sdp/healthplay/auth/SignedInFragment.java [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    }
+
+    private void deleteAccount() {
+        AuthUI.getInstance()
+                .delete(getActivity())
+                .addOnCompleteListener(getActivity(), task -> {
+                    if (task.isSuccessful()) {
+                        startActivity(new Intent(getActivity(),HomeScreenActivity.class));
+                        //finish();
+                    } else {
```
Comment (Tetardfurieux) [inline app/src/main/java/ch/epfl/sdp/healthplay/auth/SignedInFragment.java:170]: delete comment

File: app/src/main/res/navigation/my_nav.xml [hunk 4]
```diff
@@ -25,11 +25,16 @@
...
         tools:layout="@layout/fragment_game_menu" />
-    <activity
+    <!--<activity
         android:id="@+id/SignedInActivity"
         android:name="ch.epfl.sdp.healthplay.auth.SignedInActivity"
         android:label="sign_in"
-        tools:layout="@layout/signed_in_layout" />
+        tools:layout="@layout/signed_in_layout" />-->
+    <fragment
+        android:id="@+id/SignedInActivity"
```
Comment (Tetardfurieux) [inline app/src/main/res/navigation/my_nav.xml:34]: id is SignedInActivity, but it's a fragment ?? 
Please fix this

**Review by Tetardfurieux — CHANGES_REQUESTED** [review]
Good code ! Guidelines will be very helpful ! 
Remove some of the dead code, and check nav problem

**Review by Zephyr75 — APPROVED** [review]

**Review by Tetardfurieux — APPROVED** [review]

**Pull request merged**